resources/schema.gql

```diff
@@ -45,6 +45,10 @@ type Account {
 	"""
 	accountType: AccountType!
 	"""
+	Account provider
+	"""
+	accountProvider: String!
```

Comment:
Let's think why it is not enum at GQL adapter level. I understand we have plugin-like system at lower levels.